### PR TITLE
refactor: use inquirer to collect user input

### PR DESCRIPTION
**Pull request by atian25**

### CODE REVIEW

File: bin/egg-init.js [hunk 1]
```diff
@@ -59,12 +59,11 @@ co(function* () {
     log(`local template dir is ${templateDir.green}`);
   } else {
     const type = yield getType(program.type, boilerplate);
-    templateDir = yield downloadTemplates(boilerplate[type].package);
+    templateDir = yield downloadTemplates(boilerplate[ type ].package);
```
Comment (fengmk2) [inline bin/egg-init.js]: 为何要加空格？

Comment (atian25) [inline bin/egg-init.js]: 提交了

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
1.2.2

**Pull request merged**